SportHub.API/Controllers/ArticleController.cs

```diff
@@ -24,11 +25,20 @@ public class ArticleController : ControllerBase
 		}
 
 		[HttpPost(Name = "Article")]
-		public async Task<IActionResult> CreateArticleAsync([FromBody] Article article)
+		public async Task<IActionResult> CreateArticleAsync()
 		{
 			try
 			{
-				await _articlesService.CreateArticleAsync(article);
+				var file = Request.Form.Files["file"];
+
+				var articleCreateDto = JsonConvert.DeserializeObject<ArticleCreateDto>(Request.Form["article"]);
+				var article = _mapper.Map<ArticleCreateDto, Article>(articleCreateDto);
+
+				var imageCreateDto = JsonConvert.DeserializeObject<ImageCreateDto>(Request.Form["image"]);
+				var image = _mapper.Map<ImageCreateDto, Image>(imageCreateDto);
+
+				await _articlesService.CreateArticleAsync(article, image, file);
```

Comment:
file is an ASP. NET object (infrastructure object). I would suggest to create a ImageStorageService  in SportHub API that stores file and gives id or path of stored file. I suggest this because business logic should not know about and ASP. NET infrastructure code.

Reply:
This service will give the id that is assigned to Image